api/api_full.go

```diff
@@ -155,7 +155,7 @@ type FullNode interface {
 	StateMinerAvailableBalance(context.Context, address.Address, types.TipSetKey) (types.BigInt, error)
 	StateSectorPreCommitInfo(context.Context, address.Address, abi.SectorNumber, types.TipSetKey) (miner.SectorPreCommitOnChainInfo, error)
 	StatePledgeCollateral(context.Context, types.TipSetKey) (types.BigInt, error)
-	StateWaitMsg(context.Context, cid.Cid) (*MsgLookup, error)
+	StateWaitMsg(context.Context, cid.Cid, uint64, uint64) (*MsgLookup, error)
```

Comment:
We generally try naming params in functions where some of them are base-type in this file, as it's basically the main interface definition for the node API